docs/getting-started-guides/kubectl.md

```diff
@@ -0,0 +1,63 @@
+---
+---
+
+<style>
+li>.highlighter-rouge {position:relative; top:3px;}
+</style>
+
+## Overview
+
+kubectl is the command line tool you will use when interacting with kubernetes.
+
+You should use a version of kubectl that is newer than your server.  `kubectl version` will print
+the server and client versions.
+
+## Download a release
+
+Download kubectl from the [official Kubernetes releases](https://console.cloud.google.com/storage/browser/kubernetes-release/release/):
+
+On MacOS:
+
+```
+wget https://storage.googleapis.com/kubernetes-release/release/v1.4.3/bin/darwin/amd64/kubectl
+chmod +x kubectl
+mv kubectl /usr/local/bin/kubectl
+```
+
+On Linux:
+
+```
+wget https://storage.googleapis.com/kubernetes-release/release/v1.4.3/bin/linux/amd64/kubectl
```

Comment:
@pwittrock do we have a mechanism in the website that updates release numbers to the current release of K8s?  Hate to have to update this every release.